<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PullDialog</class>
 <widget class="QDialog" name="PullDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pull Resource</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QRadioButton" name="cloneRadioButton">
     <property name="text">
      <string>Clone Entire Repository</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="pullRadioButton">
     <property name="text">
      <string>Pull Selected Resource</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QCheckBox" name="recursiveCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Also pull all children of the selected resource</string>
     </property>
     <property name="text">
      <string>Pull Children</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cloneRadioButton</tabstop>
  <tabstop>pullRadioButton</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PullDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PullDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
